app/src/engineering/java/com/hedvig/app/MockActivity.kt

```diff
@@ -30,12 +29,11 @@ abstract class MockActivity : AppCompatActivity(R.layout.activity_generic_develo
         loadKoinModules(mocks)
 
         binding.root.apply {
-            setEdgeToEdgeSystemUiFlags(true)
-            doOnApplyWindowInsets { view, insets, initialState ->
-                view.updatePadding(
-                    top = initialState.paddings.top + insets.systemWindowInsetTop,
-                    bottom = initialState.paddings.bottom + insets.systemWindowInsetBottom
-                )
+            WindowCompat.setDecorFitsSystemWindows(window, false)
```

Comment:
Is there an extension for this? If no, should we write one? Something like `Window.compatSetDecorFitsSystemWindows(fits: Boolean) = WindowCompat.setDecorFitsSystemWindows(this, fits)`